src/api/java/com/minecolonies/api/crafting/RecipeStorage.java

```diff
@@ -561,14 +612,26 @@ public ResourceLocation getRecipeSource()
 
     @NotNull
     @Override
-    public List<ItemStack> getSecondaryOutputs()
+    public List<ItemStack> getSecondaryOutputs(boolean includeTools)
     {
-        return secondaryOutputs;
```

Comment:
I think we might want to separate tools from secondary outputs into a separate thing alltogether.
Either we treat them the same way as everything and just do a damage check on each item. Or we treat them differently and use two different methods.

Reply:
This is a longer discussion, but I thought about that. I'll talk to you in discord about it.